bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="broadlink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="command">
+		<item-type>String</item-type>
+		<label>IR Command</label>
+		<description>The IR command sent to the device</description>
+		<category>Text</category>
+	</channel-type>
+	<channel-type id="learning-control">
+		<item-type>String</item-type>
+		<label>Remote IR Learning Control</label>
+		<description>Instructs the Broadlink device to perform tasks related to IR code learning</description>
+		<state readOnly="false">
+			<options>
+				<option value="LEARN">Learn IR command</option>
+				<option value="CHECK">Check and save IR command</option>
+				<option value="MODIFY">Modify a previously created IR command</option>
+				<option value="DELETE">Delete IR command</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="rfcommand">
+		<item-type>String</item-type>
+		<label>RF Command</label>
+		<description>The RF command sent to the device</description>
+		<category>Text</category>
+	</channel-type>
+	<channel-type id="learning-rf-control">
+		<item-type>String</item-type>
+		<label>Remote RF Learning Control</label>
+		<description>Instructs the Broadlink device to perform tasks related to RF code learning</description>
+		<state readOnly="false">
+			<options>
+				<option value="LEARN">Learn RF command</option>
+				<option value="CHECK">Check and save RF command</option>
+				<option value="MODIFY">Modify a previously created RF command</option>
+				<option value="DELETE">Delete RF command</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<category>Humidity</category>
+		<state readOnly="true" min="0" max="100" pattern="%.2f %unit%"/>
+	</channel-type>
```

Comment:
Think this can be removed as there is a system variant for this: system.atmospheric-humidity

Reply:
Done